### DISCUSSION TITLE
misconfigured strategy or a scaling issue in a smart contract.

### DISCUSSION THREAD

**Discussion by MoshiMochiGIT** [snapshot-labs/snapshot#1730] [Q&A]
After a couple days with an active proposal we are now seeing this error for the results. "Results could not be calculated. This is often due to a misconfigured strategy or a scaling issue in a smart contract." Any ideas?

**Comment by metrox-eth** [comment]
@ChaituVR Can you look into this plz ?

**Reply by ChaituVR** [reply]
@MoshiMochiGIT Please share me the proposal URL

**Reply by MoshiMochiGIT** [reply]
> @MoshiMochiGIT Please share me the proposal URL

Appreciate the speedy response and action!

**Reply by MoshiMochiGIT** [reply]
> @MoshiMochiGIT Please share me the proposal URL

Not sure if this is the appropriate place for follow up or not but community is giving me complaints that they can't vote others are saying its taking many many tries to vote. The error from yesterday is gone and results from votes can be seen but something must still be not working as people aren't able to vote anymore. @ChatiuVR

**Reply by samwhosung** [reply]
@MoshiMochiGIT we need a link to your space or proposal to be able to help you.

**Reply by metrox-eth** [reply]
@ChaituVR 
Could be this one: 
https://snapshot.org/#/mochifactory.eth/proposal/0x865deef33331aacb52f24c7291b72769a886dff86f287ceb8998de2a28a49136
